pkgs/development/python-modules/xontrib-abbrevs/default.nix

```
@@ -0,0 +1,48 @@
+{
+  buildPythonPackage,
+  lib,
+  fetchFromGitHub,
+
+  poetry-core,
+  prompt-toolkit,
+  pytest,
+  xonsh
+}:
+
+buildPythonPackage rec {
+  name = "xontrib-abbrevs";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "xonsh";
+    repo = name;
+    rev = version;
+    sha256 = "sha256-DrZRIU5mzu8RUzm0jak/Eo12wbvWYusJpmqgIAVwe00=";
```

```suggestion
    hash = "sha256-DrZRIU5mzu8RUzm0jak/Eo12wbvWYusJpmqgIAVwe00=";
```

We use `hash`, as the SRI hash already encodes the hash type.